src/ape/api/providers.py

```diff
@@ -600,6 +600,15 @@ def set_timestamp(self, new_timestamp: int):
             int: The new timestamp.
         """
 
+    @abstractmethod
+    def mine(self, num_blocks: int):
+        """
+        Args:
+            num_blocks (int): The number of blocks allotted to mine
```

Comment:
I think we need a newline between the args and returns blocks or else the docs build funny